.spellcheck-en-custom.txt

```diff
@@ -13,3 +13,14 @@ osbuild
 qcow
 UEFI
 datadir
+customizations
+entrypoint
+fedoraproject
+filesystems
+namespaced
+untrusted
+hostname
+roadmap
+usecase
```

Comment:
I'm not a native speaker but I usually see this spelled as two words: "use case"

Reply:
Done.